sections/oxygen_pre_deployment.md

```diff
@@ -191,7 +191,7 @@ There is also a possibility of using a Data Logger while we are working with the
 Recording data using a terminal program.
 :::
 
-#### Calibration procedure
+#### Calibration protocol procedure
```

Comment:
To write both "protocol" and "procedure" is double in my view. So either or.
```suggestion
#### Calibration protocol
```

Reply:
I think the same, too repetitive, we need to decide if protocol or procedure. I will vote for procedure since ISO also uses that word for a document which includes some steps, etc.